client/platform/web-girder/views/ViewerAlert.vue

```diff
@@ -0,0 +1,60 @@
+<script lang="ts">
+import { defineComponent, ref, toRef } from '@vue/composition-api';
+
+export default defineComponent({
+  name: 'ViewerAlert',
+  setup(_, { root }) {
+    const dialog = ref(false);
+
+
+    // this will change once REST updates are merged
+    const brandData = toRef(root.$store.state.Brand, 'brandData');
+
+    return {
+      dialog,
+      brandData,
+    };
+  },
+});
+</script>
+
+<template>
+  <!-- This component is for alert messages on the Viewer Page only.
+    See NavigationBar for sitewide alert banner -->
```

Comment:
Remove comment?  I don't see anything inside this component that requires it only be rendered inside the Viewer context.

This comment is a maintenance burden because we have to remember to change it 1) if we want to use the component elsewhere or 2) if we change how NavigationBar works.